Corona-Warn-App/src/main/res/layout/contact_diary_person_list_fragment.xml

```diff
@@ -18,6 +18,8 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
+            android:paddingBottom="@dimen/spacing_huge"
+            android:clipToPadding="false"
```

Comment:
Please do it for locations too? it has the same issue

Reply:
Done 👍